test/visualization/test_visualization.cpp

```diff
@@ -110,8 +110,45 @@ TEST (PCL, PCLVisualizer_camera)
 //  cerr << "reset camera viewer pose:" << endl << viewer_pose << endl;
 }
 
-
-
+////////////////////////////////////////////////////////////////////////////////
+TEST (PCL, PCLVisualizer_getPointCloudRenderingProperties)
+{
+  PCLVisualizer::Ptr visualizer (new PCLVisualizer);
```

Comment:
This doesn't need to be a shared pointer.

Reply:
Ok, done.